verification/src/scheduler.rs

```diff
@@ -0,0 +1,60 @@
+use cron::Schedule;
+use futures::Future;
+
+use chrono::Utc;
+
+pub fn spawn_job_with_schedule<F, Fut>(
```

Comment:
may be just `scheduler::spawn_job`?
also you already take schedule as argument, so it looks like
`spawn_job_with_schedule(schedule: Scheduler...)`
vs
`spawn_job(schedule: Scheduler...) // spawn_job with (schedule,... )`

Reply:
@leviska It is not a problem here, but in general as Rust does not have function overloading that approach may not be valid. If another spawning method were there which would accept something else instead of `Schedule` (e.g. `&str`) we could not create the method `spawn_job(_: &str)`